tests/integration.rs

```diff
@@ -5490,6 +5490,39 @@ mod tests {
                     fname
                 ))
             }
+
+            fn wait_for_boot(&self) -> bool {
+                let cmd = "dir /b c:\\ | find \"Windows\"";
+                let tmo_max = 180;
+                // The timeout increase by n+n*(i+1)+..., therefore the initial
+                // interval must be small.
+                let tmo_int = 2;
+                let out = std::panic::catch_unwind(|| {
+                    match ssh_command_ip_with_auth(
```

Comment:
I think we can simplify it a bit here by removing the `catch_unwind()` that is only guarding one statement.

This should be the same: `let out = ssh_command_ip_with_auth(...).unwrap();`

Reply:
Reworked ass suggested. It was a remain from the previous more complex solution.

Thanks